server/controllers/stock/reports/stock_entry_depot.receipt.handlebars

```diff
@@ -14,6 +14,7 @@
 
     <!-- summary  -->
     <div class="row receipt-summary">
+      <h4 style="margin: 0 0 5px 0; text-align: center">{{translate 'SHIPMENT.TITLE'}}: {{entry.details.shipment_reference}}</h4>
```

Comment:
As it is, this looks somewhat out of place.  Can you make it fit the same style as the other references?

![image](https://user-images.githubusercontent.com/896472/163786799-963ace9b-4b64-4b32-a448-4b8a295e5961.png)

My thought is to move it down below and do something like:

```html
<span class="text-capitalize">{{translate 'SHIPMENT.TITLE'}}</span>: <strong>{{entry.details.shipment_reference}}</strong> <br>
```

Note that this should only be active is a `shipment_reference` exists.

Reply:
I added the #if for the shipment_reference per your suggestion.   You can see this in operation with bhima_test, by looking at the document for an older stock exit in Stock Movements.  Since no shipments were created for these stock exits, the "Shipment" label is suppressed.  But when a new stock exit is created (with or without explicit shipment), the document will include the shipment ID.